### ISSUE TITLE
No results for rpm packages when run against version 9.x of redhat/almalinux

### ISSUE THREAD

**Issue by drawks** [anchore/syft#1030]
**What happened**:

Generated SBOM only includes python package when run against the official base containers for redhat ubi 9 as well as almalinux 9

**What you expected to happen**:

Similar to version 8.x of these distros I'd expect to get a mix of rpm and python packages reported

**How to reproduce it (as minimally and precisely as possible)**:

`docker run -it --rm anchore/syft:latest redhat/ubi9`
or
`docker run -it --rm anchore/syft:latest almalinux:9`

**Anything else we need to know?**:

**Environment**:
- Output of `syft version`:
```
Application:        syft
Version:            0.46.3
JsonSchemaVersion:  3.2.3
BuildDate:          2022-05-26T06:46:34Z
GitCommit:          7cb8e1fc14a278ec5afce379623a47577aba9917
GitDescription:     v0.46.3
Platform:           linux/amd64
GoVersion:          go1.18.2
Compiler:           gc
```

**Comment by tofay** [comment]
This is fixed by https://github.com/anchore/syft/pull/1018, which is not released yet.

**Comment by tofay** [comment]
That was fixed in 0.47.0:

```bash
$ docker run -it --rm anchore/syft:latest almalinux:9
Unable to find image 'anchore/syft:latest' locally
latest: Pulling from anchore/syft
03882b0fb5fd: Pull complete
a132440c3c74: Pull complete
56b6d41be756: Pull complete
Digest: sha256:78421757823e55a1dd6f4c1e00f9754b41f714e3b2f2cd8aed04279fa84e75c7
Status: Downloaded newer image for anchore/syft:latest
 ✔ Parsed image
 ✔ Cataloged packages      [153 packages]
NAME                         VERSION                        TYPE
acl                          2.3.1-3.el9                    rpm
almalinux-gpg-keys           9.0-3.el9                      rpm
almalinux-release            9.0-3.el9                      rpm
...
```
```bash
$ docker run -it --rm anchore/syft:latest redhat/ubi9
 ✔ Parsed image
 ✔ Cataloged packages      [202 packages]

NAME                                    VERSION                        TYPE
PyGObject                               3.40.1                         python
PySocks                                 1.7.1                          python
acl                                     2.3.1-3.el9                    rpm
alternatives                            1.20-2.el9                     rpm
audit-libs                              3.0.7-101.el9_0.2              rpm
basesystem                              11-13.el9                      rpm
bash                                    5.1.8-4.el9                    rpm
...
```
